themes/wilson_theme_starterkit/src/scss/styles.scss

```diff
@@ -20,3 +20,22 @@
 // Import Tailwind components & utilities.
 @tailwind components;
 @tailwind utilities;
+
+// Custom CSS variables.
+:root {
```

Comment:
Another question: a few of the themes I've worked on have Bower/Susy included and as far as I can tell, that's _only_ to use the grid structure... I'm wondering if this is the kind of thing we could lift outta Wilson and apply to some of our other themes to replace Susy?

(This is a very nity piece of CSS. Too much maths for me :joy: but I understand it!)

Reply:
Obviously we'd have to do more than "just copy and paste it in" :D I'm mostly thinking out loud.

Reply:
Since I've been in Tailwind land in the last couple of projects, the prospect of working on traditional CSS + Susy harms my vibe. I'm sure with enough time and cups of tea, we could retro-fit CSS grid layouts in to old themes but I think it would be more beneficial to wait for a good reason to create a new theme and base it on modern Tailwind which handles grid nicely. 

You wouldn't normally need to do the weirdness with CSS variables like I did here, this was because I was trying to find the most compact way of providing utility CSS classes for asymmetric grid templates, and I got carried away.